src/main/resources/db/migration/V18_01__add_veileder_historikk.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE VEILEDER_HISTORIKK (
+  id                        SERIAL             PRIMARY KEY,
+  created_at                TIMESTAMPTZ        NOT NULL,
+  uuid                      CHAR(36)           NOT NULL UNIQUE,
+  person_oversikt_status_id INTEGER REFERENCES PERSON_OVERSIKT_STATUS (id) ON DELETE CASCADE,
+  tildelt_dato              DATE,
+  tildelt_veileder          VARCHAR(7)         NOT NULL,
+  tildelt_enhet             VARCHAR(4)         NOT NULL,
+  tildelt_av                VARCHAR(7)
```

Comment:
Kunne man eventuelt laget en konstant vi lagrer her? 🤔 Blir kanskje dumt om vi ønsker å typesikre verdien. Kjedelig å for alltid måtte tenke på at denne kan være `NULL` 🥲

Reply:
Ja, kanskje bruke "X000000"?